frontend/src/store/secret.js

```diff
@@ -145,7 +149,7 @@ export const useSecretStore = defineStore('secret', () => {
     dnsSecretList,
     dnsSecretsByProviderKind,
     getCloudProviderSecretByName,
-    infrastructureSecretsByCloudProfileName,
+    infrastructureSecretsByCloudProviderKind,
```

Comment:
same here. rename to ...ProviderType

Reply:
I replaced all occurrences of providerKind with providerType. I wanted to doi this in a separate PR at first. But as I started renaming the things I touched in this PR it felt like it makes no sense stopping there.